tldextract/__init__.py

```diff
@@ -1,6 +1,20 @@
 """Export tldextract's public interface."""
 
-import pkg_resources
+import os
+
+try:
+    import pkg_resources
+except ImportError:
+    class pkg_resources(object):  # pylint: disable=invalid-name
+
+        """Fake pkg_resources interface which falls back to getting resources
+        inside `tldextract`'s directory.
+        """
+        @classmethod
+        def resource_stream(cls, _, resource_name):
```

Comment:
In this file, the fake fallback is only used for `pkg_resources.get_distribution`, a few lines down (outside of the range I can comment on). This fake doesn't define that method. Does this trade an `ImportError` for an `AttributeError`?